iotedgehubdev/cli.py

```diff
@@ -261,6 +267,10 @@ def start(inputs, port, deployment, verbose, host, environment, edgehub_image_ve
             if not verbose:
                 output.info('IoT Edge Simulator has been started in solution mode.')
         else:
+            if edge_runtime_version is not None:
+                if re.match(r'^(1\.0)|(1\.1)', edge_runtime_version) is None:
```

Comment:
not going to worry about the long list at https://mcr.microsoft.com/v2/azureiotedge-hub/tags/list ?

Reply:
agree, i think we should allow any value here. iotedgedev and vscode can domore strict validation because they pull the agent image too.

---
In reply to: [600738765](https://github.com/Azure/iotedgehubdev/pull/320#discussion_r600738765) [](ancestors = 600738765)

Reply:
In the discussion email thread we had agreed to restrict versions to 1.0.x and 1.1.x exclusively, hence the code as shown.

Reply:
Agreed with @cindydeng1998 and @marianan , limiting to 1.0 and 1.1 only may limit most of our customers using this via cicd